lib/yrewrite/seo.php

```diff
@@ -102,6 +102,46 @@ public function getTags()
         return implode("\n", $tags);
     }
 
+    /** @deprecated use getTags instead */
+    public function getTitleTag()
+    {
+        return '<title>'.rex_escape(strip_tags($this->getTitle())).'</title>'; // lang="de"
+    }
+
+    /** @deprecated use getTags instead */
+    public function getDescriptionTag()
+    {
+        return '<meta name="description" content="'.rex_escape(strip_tags($this->getDescription())).'">'; //  lang="de"
+    }
+
+    /** @deprecated use getTags instead */
+    public function getCanonicalUrlTag()
+    {
+        return '<link rel="canonical" href="'.rex_escape($this->getCanonicalUrl()).'" />';
+    }
+
+    /** @deprecated use getTags instead */
+    public function getRobotsTag()
+    {
+        if ($this->article->getValue(self::$meta_index_field) == 1 || ($this->article->getValue(self::$meta_index_field) == 0 && $this->article->isOnline())) {
+            return '<meta name="robots" content="index, follow">';
+        } else {
+            return '<meta name="robots" content="noindex, nofollow">';
+        }
+    }
```

Comment:
In den Artikel-Einstellungen gibt's eigentlich auch noch die Variante noindex, follow:

![image](https://user-images.githubusercontent.com/3855487/163162233-6ab6f4c5-b0d1-4cfb-87c9-9ce5d57f99d1.png)

Reply:
> In den Artikel-Einstellungen gibt's eigentlich auch noch die Variante noindex, follow:

Ich habe die alten Methoden unverändert wieder aufgenommen. Anpassungen daran sollte ein anderer PR machen.

Reply:
Naja gut, das lohnt ja dann nicht mehr, wenn sie im nächsten Step eh rausfliegen. Danke für die Erklärung!